### PR TITLE
ReText translated to Croatian

### PR DESCRIPTION
**Pull request by krunose**
Translation of user interface to Croatian.

### CODE REVIEW

File: locale/retext_hr.ts [hunk 1]
```diff
@@ -0,0 +1,624 @@
...
+    </message>
+    <message>
+        <location filename="window.py" line="1156"/>
+        <source>Simple but powerful editor for Markdown and reStructuredText</source>
+        <translation>Jednostavan ali moćan uređivač teksta pisanoga Markdownom i reStructuredTextom</translation>
+    </message>
+    <message>
+        <location filename="window.py" line="1156"/>
+        <source>Author: Dmitry Shachnev, 2011</source>
+        <translation>Autor: Dmitry Shachnev (Dmitry Shachnev), 2011</translation>
```
Comment (mitya57) [inline locale/retext_hr.ts]: Why did you duplicate my name? :)

File: locale/retext_hr.ts [hunk 2]
```diff
@@ -0,0 +1,624 @@
...
+    </message>
+    <message>
+        <location filename="window.py" line="211"/>
+        <source>About ReText</source>
+        <translation>O ReTextu</translation>
+    </message>
+    <message>
+        <location filename="window.py" line="214"/>
+        <source>About Qt</source>
+        <translation>O QT-u</translation>
```
Comment (mitya57) [inline locale/retext_hr.ts]: Please change QT → Qt.

File: locale/retext_hr.ts (hunk 1 above)
Comment (krunose) [reply locale/retext_hr.ts]: Used retext_sr@latin.ts as template :D

**Comment by mitya57** [comment]
Thank you again and enjoy your first pull request merged :)

**Comment by krunose** [comment]
I will :) thanks!

**Pull request merged**